Contributing.md

```diff
@@ -137,7 +150,7 @@ When you're finished with your work and are ready to have it merged into the mai
 6. Add a title to your pull request, and describe all the changes you're making.
```

Comment:
Line 109 is missing a colon and should start with something like `**Important**:`